emacs: bump elisp packages

jian-lin opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

- emacs-overlay commit: c3e2ca282d91c806f1c10ae9387c9631241f21f7
- new failed toplevel builds:
  - [X] emacs.pkgs.procress
  - [x] emacs.pkgs.blue: [upstream report][1]
  - [x] emacs.pkgs.elisp-depmap: [upstream report][2]
  - [ ] emacs.pkgs.sdml-mode

[1]: https://codeberg.org/lapislazuli/blue.el/issues/3
[2]: https://github.com/mtekman/elisp-depmap.el/issues/7

Closes: #449908

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [X] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [X] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

jian-lin committed:
nongnu-packages: updated 2025-10-10 (from overlay)

jian-lin committed:
melpa-packages: updated 2025-10-10 (from overlay)

jian-lin committed:
elpa-devel-packages: updated 2025-10-10 (from overlay)

jian-lin committed:
elpa-packages: updated 2025-10-10 (from overlay)

jian-lin committed:
nongnu-devel-packages: updated 2025-10-10 (from overlay)

jian-lin committed:
emacs.pkgs.procress: fix build

panchoh reviewed (approved):
Verified following the established [procedure](https://github.com/NixOS/nixpkgs/pull/412569#issuecomment-2924461614), more or less the same way¹ I did [the last time](https://github.com/NixOS/nixpkgs/pull/435408#issuecomment-3227344730).

Everything checks out, ofc, including the removal of the newly failing package `procress` that you mentioned.

👍

¹: I used `git-worktree` this time.

jian-lin committed:
emacs.pkgs.blue: 20251009.1324 -> 20251010.1137

This bump fixes build failure of missing dependencies.

https://codeberg.org/lapislazuli/blue.el/issues/3
